_posts/2020-11-11--kinesis-streams-part-2.md

```diff
@@ -0,0 +1,222 @@
+---
+layout: post
+title: Mastering AWS Kinesis Data Streams, Part 2
+author: anahitpo
+excerpt: >
+  By now we have gone in quite some details when it comes to writing your dat to a Kinesis stream. How about making use of that data, aka reading it from a stream?
+tags:
+- AWS
+- Data streaming
+- Lambda
+- Serverless
+- Best practices
+---
+
+DO NOT MERGE YET!!! :)
+
+
+This is the second part of the post about AWS _Kinesis Data Streams_. Part 1 is [here](https://dev.solita.fi/2020/05/28/kinesis-streams-part-1.html).
```

Comment:
Good!

Reply:
Don't forget to update part 1 with a link to this part 2!